reservations/views.py

```diff
@@ -1,3 +1,24 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from .models import Reservation
+from .serializers import ReservationSerializer
+
+class ReservationListView(APIView):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        serializer = ReservationSerializer(data=data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=201)
```

Comment:
마지막으로 status code 역시 rest framwork에서 제공하는 status code를 사용해주세요!
힌트) `from rest_framework import status`
```suggestion
            return Response(serializer.data, status=201)
```